.gitignore

```diff
@@ -2,4 +2,5 @@ out
 node_modules
 .vscode-test/
 *.vsix
+*.lock
```

Comment:
why? yarn.lock is meant to be checked out in version control

Reply:
och, I thought it might just be a mistake. Didn't know lock files were sensible to be included

Reply:
yeah, the purpose of yarn.lock is to "lock" the dependency tree to ensure the build is reproducible.

Unfortunate name, but there have been worse ones in this space (google "npm shrinkwrap" for an example 😅)